x-pack/plugins/fleet/public/components/agent_enrollment_flyout/instructions.tsx

```diff
@@ -42,11 +40,16 @@ export const Instructions = (props: InstructionProps) => {
     refreshAgentPolicies,
   } = props;
   const fleetStatus = useFleetStatus();
-  const { isUnhealthy: isFleetServerUnhealthy } = useFleetServerUnhealthy();
+  const REFRESH_INTERVAL = 10 * 1000;
```

Comment:
there is no way to fix the root problem and refresh the status when fleet server is added? or at least only once when the  flyout is open?

Reply:
For some reason when the flyout is first open the fleetStatus is not ready, so you have to reload the page to get it. I couldn't find a solution that refreshes the state, I tried in several ways using `useEffect` but they mostly ended up int `too many rerenders` errors.